sys/dev/ce/if_ce.c

```diff
@@ -2630,14 +2639,22 @@ MODULE_DEPEND (ce, sppp, 1, 1, 1);
 #endif
 #ifdef KLD_MODULE
 DRIVER_MODULE (cemod, pci, ce_driver, ce_devclass, ce_modevent, NULL);
+MODULE_PNP_INFO("U16:vendor; U16:device", pci, cemod, ce_devs,
+     sizeof(ce_devs[0]), nitems(ce_devs));
 #else
 DRIVER_MODULE (ce, pci, ce_driver, ce_devclass, ce_modevent, NULL);
+MODULE_PNP_INFO("U16:vendor; U16:device", pci, ce, ce_devs,
+     sizeof(ce_devs[0]), nitems(ce_devs));
 #endif
 #else /* if __FreeBSD_version < 500000*/
 #ifdef NETGRAPH
```

Comment:
the next two can be omitted. In fact, all this code for FreeBSD < 5.0 should be removed.